### DISCUSSION TITLE
Equipment Type: Tractor Beam

### DISCUSSION THREAD

**Discussion by zalexville** [tsunamayo/Starship-EVO#4693] [Suggestions]
**Tractor Beam**

The basic premise is a beam type, like the repair beam or beam weapon, that could grab and move another entity. It could move that entity around and bring it closer or further to the tractor beam emitter. I am not sure how to work size and weight limitations, but it would need to scale with the emitter size. For example, a 1m tractor could move a ship up to 1ton, 4m could move 4tons, 10m could move 10tons. I don't see why you also cant use multiple tractors to grab the same entity, that way tractor beams can be used by multiple ships to carry a much heavier entity or pass it along to another tractor emitter. Tractor cannot lock onto shielded targets. 

I am not sure how feasible something like this would be to build into the game but if it could be done then moving cargo, ships in a hangar, and even locking things into place without the need of a maglock could be achieved potentially. A handheld tractor beam might be useful too for moving crates and stuff around (kinda like what Star Citizen is doing to be fair). 

![image](https://user-images.githubusercontent.com/65626266/167662783-7567cee1-f443-4cc8-919c-3b2df99a264f.png)

**Comment by Briaireous** [comment]
yeah, that tractor add-on for the multitool in SC is hella handy

**Comment by Malus-Draco** [comment]
its exactly what we need for moving things around manually, im taking extreme measures at the moment with gravity and maglocks
